cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -214,7 +214,8 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
         query("ALTER DATABASE `topology-example` SET TOPOLOGY 1 PRIMARY 0 SECONDARIES", ResultAssertions(r => {
           assertStats(r, systemUpdates = 1)
         })) {
-          statsOnlyResultTable()
+
+//          statsOnlyResultTable()
```

Comment:
why not remove it fully if you don't want to run the query?

Reply:
WIP haven't figured out how to "fake" a table here yet